src/main/java/bbd/jportal2/generators/FreeMarker/FreeMarker.java

```diff
@@ -126,6 +127,7 @@ private void GenerateSingleFTLFile(String templateBaseDir, String generatorName,
         //Set up FreeMarker object maps
         java.util.Map<String, Object> root = new HashMap<>();
         root.put("database", database);
+        root.put("table", database.tables.get(0));
```

Comment:
I was going to say don't do this, but I understand why. Good idea.

Reply:
It's a bit of a stop-gap. We'll have to do it better when we start fleshing out the "project" based generation, where there is more than one table in tables. But, this didn't break any tests, so should be fine? :)